.github/workflows/perl-package-build.yml

```diff
@@ -5,40 +5,38 @@ on: [push, pull_request]
 jobs:
   test-perl-package:
     runs-on: ubuntu-latest
+    container:
+      image: perldocker/perl-tester:latest
 
     steps:
-      - name: Setup perl
-        uses: shogo82148/actions-setup-perl@v1
-        with:
-          perl-version: ${{ inputs.perl-version }}
-
-      - name: Install Dist::Zilla
-        shell: bash
-        run: |
-          cpanm --quiet Dist::Zilla Archive::Tar::Wrapper
+      - name: Perl version
+        run: perl -V
 
       - name: Install system dependencies
         shell: bash
-        run: |
-          sudo apt-get update
-          sudo apt-get install -y lz4
+        run: apt-get install --no-install-recommends -y lz4
 
       - name: Install one-off Bystro dependencies
         shell: bash
         run: |
-          cpanm --quiet https://github.com/bystrogenomics/msgpack-perl.git
-          cpanm --quiet --notest MouseX::Getopt
-          cpanm --quiet DBD::mysql@4.051
+          cpm install -g https://github.com/bystrogenomics/msgpack-perl.git
+          cpm install -g --no-test MouseX::Getopt
+          cpm install -g DBD::mysql@4.051
           git clone --depth 1 --recurse-submodules https://github.com/salortiz/LMDB_File.git \
             && cd LMDB_File \
             && cpanm --quiet .
```

Comment:
non-blocking minor note, but do we need to continue to use cpanm here?

Reply:
The only cpanm we need to use is the one with LMDB_File. The one you point out could be changed.